docs/Makefile

```diff
@@ -0,0 +1,6 @@
+.PHONY:
+all: $(patsubst %.puml,%.svg,$(wildcard *.puml))
+
+%.svg: %.puml
+	test -n "$$(podman images -q -f reference=localhost/plantuml)" || podman build -f Dockerfile.plantuml . -t plantuml
+	podman run --rm -v $$(pwd):/mnt:z plantuml plantuml -tsvg $<
```

Comment:
Never knew about the `:z` option before.  I feel like it probably took a lot of cursing at SELinux to figure that out.

Reply:
Yeah, for sure, the first dozen times (though I still often forget). They do actually document it if you think to look though: https://github.com/containers/podman/blob/main/troubleshooting.md#2-cant-use-volume-mount-get-permission-denied